lib/airtable/table.rb

```diff
@@ -71,8 +93,8 @@ def result_attributes(res)
     end
 
     def worksheet_url
-      "/#{app_token}/#{URI.encode(worksheet_name)}"
+      "/#{app_token}/#{CGI.escape(worksheet_name)}"
```

Comment:
@syrnick a [combination of](http://webcache.googleusercontent.com/search?q=cache:LtYu7L5Rm48J:ruby-core.ruby-lang.narkive.com/0XKxdn7l/ruby-core-29293-uri-un-escape-deprecated+&cd=7&hl=en&ct=clnk&gl=ca) `CGI.escape` for the base and `URI.encode_www_form` for the params.